xml/System.Windows.Forms/ContainerControl.xml

```diff
@@ -1195,10 +1195,11 @@
         <Parameter Name="updateContainerSize" Type="System.Boolean" Index="2" FrameworkAlternate="windowsdesktop-7.0" />
       </Parameters>
       <Docs>
-        <param name="xScaleFactor">To be added.</param>
-        <param name="yScaleFactor">To be added.</param>
-        <param name="updateContainerSize">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="xScaleFactor">The scale factor to be applied on width of the property being scaled.</param>
+        <param name="yScaleFactor">The scale factor to be applied on height of the property being scaled.</param>
+        <param name="updateContainerSize">
+          <see langword="true" /> to resize of the container control along with properties being scaled; otherwise, <see langword="false" />.</param>
```

Comment:
```suggestion
          <see langword="true" /> to resize the container control along with properties being scaled; otherwise, <see langword="false" />.</param>
```

Reply:
👍